### PR TITLE
Added Multiplexers and Demultiplexers

### PR DESCRIPTION
**Pull request by mohitgujarathi14**
- Added multiplexer IC74152, IC74151A ,IC74153
- Added tests and examples for IC74152, IC74151A ,IC74153
- Added demultiplexer ICs 74138 ,74139 ,74155 ,74156 
- Added tests and examples for IC74156, IC74155, IC74139 ,IC74138
- Deleted series_7400_tests.py since ics_tests.py already existed
- Updated ics_tests with complete tests
- Updated .travis.yml

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -20,7 +20,6 @@ script:
  - nosetests -v sequential_tests.py
  - nosetests -v counters_tests.py
  - nosetests -v series_4000_test.py
- - nosetests -v series_7400_tests.py
  - nosetests -v source_tests.py
  - nosetests3 -v gates_tests.py
  - nosetests3 -v ics_tests.py
```
Comment (rajat974) [inline .travis.yml:25]: It will be better to change the `ics_tests.py` to `series_7400_tests.py`

**Comment by rajat974** [comment]
tests failed because you didn't import MUX , and you can run `nosetests` and `nosetests3` locally to check the tests before pushing.

**Comment by mohitgujarathi14** [comment]
Ahh.. Locally I did import MUX and the tests had ran successfully.

**Comment by jayrambhia** [comment]
This seems really good. Add some examples too.

File: BinPy/ic/series_7400.py [hunk 2]
```diff
@@ -1604,6 +1605,53 @@ def run(self):
...
+        4	D1
+        5	D0
+        6	Output W
+        7	Ground
+        8	select line C
+        9	select line B
+        10	select line A
+        11	D7
+        12	D6
+        13  D5
```
Comment (rajat974) [inline BinPy/ic/series_7400.py:1629]: indentation

File: BinPy/ic/series_7400.py [hunk 3]
```diff
@@ -2242,3 +2290,341 @@ def run(self):
...
+        6	1C0
+        7	1Y - OUTPUT1
+        8	Ground
+        9	2Y - OUTPUT2
+        10	2C0
+        11	2C1
+        12	2C2
+        13	2C3
+        14  Select line A
+        15  Strobe2
```
Comment (rajat974) [inline BinPy/ic/series_7400.py:2317]: indentation

Comment (mohitgujarathi14) [inline BinPy/ic/series_7400.py:2317]: Strange! When I go to edit it, it shows proper indentation

**Comment by mohitgujarathi14** [comment]
@jayrambhia please see #162

**Pull request closed without merging**